src/validators/mac-address

```diff
@@ -15,12 +15,15 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import re
-from sys import exit, argv
+import sys
```

Comment:
Why pull in everything when we can only pull in two functions?

Reply:
According to some python gurus, rhis shouldn't make any difference in the execution time or memory usage, the whole module is loaded in each case. It does have benefits, to prevent function name clashes and make the code terser.